modules/godot/databags/input_databag.cpp

```diff
@@ -204,5 +204,5 @@ void InputDatabag::add_input_event(const Ref<InputEvent> &p_event) {
 		}
 	}
 
-	ERR_PRINT(String("This input event type is not recognized, please open an issue so we can support it: ") + Variant(p_event));
```

Comment:
Weird that it's not automatically casted anymore. Can you please use `String(Variant(p_event))` or `Variant(p_event).stringify()` instead?

Reply:
It was implicitly cast, but it appears there was since an overload of the cast that was added making it ambiguous between two forms. I'll try these and see if they correctly work, as I can't remember what the two forms are.